src/server/game/DataStores/DBCEnums.h

```diff
@@ -916,56 +975,70 @@ enum class CurveInterpolationMode : uint8
 
 enum Difficulty : uint8
 {
-    DIFFICULTY_NONE                 = 0,
-    DIFFICULTY_NORMAL               = 1,
-    DIFFICULTY_HEROIC               = 2,
-    DIFFICULTY_10_N                 = 3,
-    DIFFICULTY_25_N                 = 4,
-    DIFFICULTY_10_HC                = 5,
-    DIFFICULTY_25_HC                = 6,
-    DIFFICULTY_LFR                  = 7,
-    DIFFICULTY_MYTHIC_KEYSTONE      = 8,
-    DIFFICULTY_40                   = 9,
-    DIFFICULTY_3_MAN_SCENARIO_HC    = 11,
-    DIFFICULTY_3_MAN_SCENARIO_N     = 12,
-    DIFFICULTY_NORMAL_RAID          = 14,
-    DIFFICULTY_HEROIC_RAID          = 15,
-    DIFFICULTY_MYTHIC_RAID          = 16,
-    DIFFICULTY_LFR_NEW              = 17,
-    DIFFICULTY_EVENT_RAID           = 18,
-    DIFFICULTY_EVENT_DUNGEON        = 19,
-    DIFFICULTY_EVENT_SCENARIO       = 20,
-    DIFFICULTY_MYTHIC               = 23,
-    DIFFICULTY_TIMEWALKING          = 24,
-    DIFFICULTY_WORLD_PVP_SCENARIO   = 25,
-    DIFFICULTY_5_MAN_SCENARIO_N     = 26,
-    DIFFICULTY_20_MAN_SCENARIO_N    = 27,
-    DIFFICULTY_PVEVP_SCENARIO       = 29,
-    DIFFICULTY_EVENT_SCENARIO_6     = 30,
-    DIFFICULTY_WORLD_PVP_SCENARIO_2 = 32,
-    DIFFICULTY_TIMEWALKING_RAID     = 33,
-    DIFFICULTY_PVP                  = 34,
-    DIFFICULTY_NORMAL_ISLAND        = 38,
-    DIFFICULTY_HEROIC_ISLAND        = 39,
-    DIFFICULTY_MYTHIC_ISLAND        = 40,
-    DIFFICULTY_PVP_ISLAND           = 45,
-    DIFFICULTY_NORMAL_WARFRONT      = 147,
-    DIFFICULTY_HEROIC_WARFRONT      = 149,
-    DIFFICULTY_LFR_15TH_ANNIVERSARY = 151,
-    DIFFICULTY_VISIONS_OF_NZOTH     = 152,
-    DIFFICULTY_TEEMING_ISLAND       = 153
+    DIFFICULTY_NONE                         = 0,
+    DIFFICULTY_NORMAL                       = 1,    // 1 (Party Dungeon),   MinPlayers:  5, MaxPlayers:  5
+    DIFFICULTY_HEROIC                       = 2,    // 1 (Party Dungeon),   MinPlayers:  5, MaxPlayers:  5
+    DIFFICULTY_10_NORMAL                    = 3,    // 2 (Raid Dungeon),    MinPlayers: 10, MaxPlayers: 10
+    DIFFICULTY_25_NORMAL                    = 4,    // 2 (Raid Dungeon),    MinPlayers: 25, MaxPlayers: 25
+    DIFFICULTY_10_HEROIC                    = 5,    // 2 (Raid Dungeon),    MinPlayers: 10, MaxPlayers: 10
+    DIFFICULTY_25_HEROIC                    = 6,    // 2 (Raid Dungeon),    MinPlayers: 25, MaxPlayers: 25
+    DIFFICULTY_LFR                          = 7,    // 2 (Raid Dungeon),    MinPlayers: 25, MaxPlayers: 25
```

Comment:
at this point, I would rename DIFFICULTY_LFR to DIFFICULTY_LFR_OLD, since it was the one used for Flexible raid when it was initially introduced in 5.4, and DIFFICULTY_LFR_NEW to DIFFICULTY_LFR since it's the standard

Reply:
this enum has been pulled from interface code.

Reply:
> this enum has been pulled from interface code.

https://wago.tools/db2/Difficulty

Reply:
> this enum has been pulled from interface code.

more like db2, either way, both are called Looking for Raid, so it's more than reasonable to rename them as I indicated